cmd/create/cluster/cmd.go

```diff
@@ -256,6 +256,16 @@ func run(cmd *cobra.Command, _ []string) {
 
 	// Get cluster name
 	clusterName := args.clusterName
+
+	if clusterName == "" && !interactive.Enabled() {
+		if len(argv) != 1 {
```

Comment:
We don't actually use `argv` for the name:
```
$ rosa create cluster foo
E: Cluster name must consist of no more than 15 lowercase alphanumeric characters or '-', start with a letter, and end with an alphanumeric character.
```
So I think that check can go away entirely, or we can change the code to determine what to use:
```go
clusterName := args.clusterName
if clusterName == "" {
  clusterName = argv[0]
}
if clusterName == "" && !interactive.Enabled() {
  interacive.Enable()
  reporter.Infof("Enabling interactive mode")
}
```

Reply:
i removed the argv since it will be nil if no values are specified.